src/solvespace.h

```diff
@@ -621,7 +621,7 @@ class SolveSpaceUI {
     std::string MmToString(double v, bool editable=false);
     std::string MmToStringSI(double v, int dim = 0);
     std::string DegreeToString(double v);
-    double ExprToMm(Expr *e);
+    double NonConstraintExprToMm(Expr *e);
```

Comment:
I chose to rename this function, since using it to get the value of a constraint's expression in base Mm is a bad idea, considering scale factor stapled to each expression in a constraint, which encodes the scale factor that constraint was "saved" under, which itself was necessarily to correctly render constraint labels while applying the right scalings. 

By renaming the function, I'm hoping to force callers to declare compliance with this invariant